.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement'
+assignees: ''
+---
+<!--
+Thank you for suggesting an improvement to OpenRewrite!
+Feel free to delete any sections that don't apply to your request.
+-->
+## What problem are you trying to solve?
+
+## Describe the solution you'd like
+
+## Have you considered any alternatives or workarounds?
+
+## Additional context
+<!-- Any thoughts that come to mind, or similar items on the backlog already. -->
+
+## Are you interested in contributing this feature to OpenRewrite?
+<!-- Indicate if this is something you would like to work on, and how we can best support you in doing so. -->
```

Comment:
I like the way this is phrased 👍

Reply:
Thanks! Yes I figured it drives home that we're open to contributions; should we through in a link to contributing.md?

Reply:
Ya that makes sense to me ^_^